source/settings/core.rst

```diff
@@ -4831,7 +4831,7 @@ Client certificate private key used in outgoing SSL connections.
 
 Example Setting:
 
-   ssl_client_cert = </etc/dovecot/dovecot-client.crt
+   ssl_client_cert = </etc/dovecot/dovecot-client.key
```

Comment:
Why?.. key file makes more sense for ssl_client_key.

Reply:
Yeah, didn't have the whole fix. This is in ssl_client_key section and the example is for cert.